app/renderers/shopify/redcarpet_html_renderer.rb

```diff
@@ -11,6 +11,7 @@ def initialize(options: {}, extensions: {})
     end
 
     def render(content)
+      return '' if content.nil?
```

Comment:
The ole nil game. Nice catch that sometimes this will return nil blowing everything up in the render method below.